visualization/app/codeCharta/ui/ribbonBar/artificialIntelligence/suspiciousMetrics/suspiciousMetrics.component.html

```diff
@@ -22,6 +22,11 @@
 					: "Unsuspicious Metrics"
 			}}
 		</div>
+		<div class="suspicious-metric">
+			This feature compares the values of different metrics with metric values of 241 Open Source Java projects. Based on this data
```

Comment:
```suggestion
			This feature compares the values of different metrics with metric values of several Open Source Java projects. Based on this data
```
I would omit the exact number, in case we need to change it in the future.

Reply:
I would not, the benchmark data is fixed and it is nice to show this result :-)